index.bs

```diff
@@ -5796,8 +5796,8 @@ of specifications using Web IDL might want to obtain a sequence of
 </div>
 
 There is no way to represent a constant {{DOMString}}
-value in IDL, although {{DOMString}} [=dictionary member=]
-and [=optional argument|operation optional argument=] default values
+value in IDL, although enumeration-typed [=dictionary member=] [=dictionary member/default values=]
```

Comment:
These changes seem wrong; it should still be DOMString, ByteString, USVString, etc.

Reply:
Of course, done.